### ISSUE TITLE
HAMX v7.3.0 can not run kernel-qemu

### ISSUE THREAD

**Issue by lxf-sole** [intel/haxm#97]
I installed HAMX v7.3.0 and start an avd using kernel-qemu, but the avd can not start normally. The log repeatly shows : "emulator: Trying to vcpu execute at eip:8b68fb".
but the avd can normally start if using HAMX v7.2.0 or using kernel-ranchu

By the way, my need is to start an avd without the floating toolbar on the right and I couldn't find a way to hide the floating toolbar, so I have to use the SDK tools(emulator.exe) v24.4.1. But it seems that SDK tools v24.4.1 can only start kernel-qemu and can not start kernel-ranchu, and I found the above problem. If there is a way to hide the floating toolbar, please tell me.

**Comment by raphaelning** [comment]
The `Trying to vcpu execute at eip:` message indicates that your CPU does not support a Intel VT-x feature known as _Unrestricted Guest_ (UG). What is your CPU model?

Unfortunately, although there is still a code path in HAXM and Android Emulator for non-UG hosts, it is no longer tested, at least not by the HAXM team anymore. Nor do we test or support Android Emulator 24.4.1 or the non-Ranchu Android kernel, both of which are very old. If HAXM 7.2.0 still works for you, my advice is to stick to it.

> my need is to start an avd without the floating toolbar on the right and I couldn't find a way to hide the floating toolbar

I see. You could try to submit a feature request for Android Emulator:

https://developer.android.com/studio/report-bugs#emulator-bugs
http://issuetracker.google.com/issues?q=componentid:192727%2B

**Comment by lxf-sole** [comment]
To: raphaelning
Thank you for your answer.

CPU:  Intel(R) Core(TM) i5-6500 CPU @3.20GHz 3.19 GHz
or
CPU:  Intel(R) Core(TM) i7-6700 CPU @3.40GHz 3.40GHz

is this two CPU very old?

**Comment by raphaelning** [comment]
Not at all! Both models definitely support UG. I was referring to pre-2011 models which are almost extinct now.

> The `Trying to vcpu execute at eip:` message indicates that your CPU does not support a Intel VT-x feature known as Unrestricted Guest (UG).

Sorry, I overlooked another code path that can print this message, and that code path only exists in the old/obsolete Android Emulator code base (a.k.a. _qemu1_, whereas the current code base is _qemu2_). I suspect that only _qemu1_ preserves the old GUI (no toolbar). Moreover, _qemu1_ does not take advantage of UG at all, or other great features we developed in recent years, so I feel sorry for your powerful CPU being wasted.

We don't have the bandwidth to support _qemu1_. It would be great if you could switch to the current Android Emulator (_qemu2_) soon. Have you submitted a feature request to Google?
